sycl/doc/EnvironmentVariables.md

```diff
@@ -152,7 +152,6 @@ Note that conflicting configuration tuples in the same list will favor the last
 
 | Environment variable | Values | Description |
 | -------------------- | ------ | ----------- |
-| `SYCL_ENABLE_PCI` | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. |
```

Comment:
Since this is described not in experimental env vars section, I think that we keep it for now, say it is deprecated, and is ignored, and will be removed.

Reply:
cool, changing @smaslov-intel

Reply:
@smaslov-intel please check latest commit, https://github.com/intel/llvm/pull/10681/commits/5937848f07ed8d8ed013f227da7d60458ba40a07, to confirm statement added is correct.